frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/prefill-vm-template-state-update.ts

```diff
@@ -164,9 +165,17 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
       const provisionSourceDetails = ProvisionSource.getProvisionSourceDetails(template, {
```

Comment:
so maybe this isnt neccessary to do it twice two lines above

`vmSettingsUpdate[VMSettingsField.PROVISION_SOURCE_TYPE] = {`

Reply:
removed the redundancy